sql/CMakeLists.txt

```diff
@@ -804,7 +804,7 @@ TARGET_LINK_LIBRARIES(sql_main ${MYSQLD_STATIC_PLUGIN_LIBS}
 # sql/immutable_string.h uses
 # google::protobuf::io::CodedOutputStream::WriteVarint64ToArray
 # which may or may not be inlined.
-TARGET_LINK_LIBRARIES(sql_main ${PROTOBUF_LITE_LIBRARY})
+#TARGET_LINK_LIBRARIES(sql_main ${PROTOBUF_LITE_LIBRARY}) have been removed because PXB doesn't directly use immutable string and we don't want a protobuf.so dependency to use PXB
```

Comment:
has been removed

Reply:
Now that you know the wording and reason, can you please update the commit message as well?